### ISSUE TITLE
[CONTENT] Add adventure with only visible output screens.

### ISSUE THREAD

**Issue by MarleenGilsing** [hedyorg/hedy#4007]
I would like to add adventures that have a hidden inputcode, but you can still run it and see the output. The student has to recreate the code. 
This could also be done in a quiz form, so the students get like 5 codes increasing in difficulty to recreate.

 I could make this, don't know how yet, but I'll make it an issue so I don;t forget about it.

**Comment by Felienne** [comment]
Is this the same as (or similar to) #1936 but more generic?

**Comment by AnneliesVlaar** [comment]
@Felienne and @MarleenGilsing is this something we now can do in the workbooks?

**Comment by MarleenGilsing** [comment]
No, because I would like the program to be interactive (I have used the word output wrong). 
So an invisible program you could run, and then try to recreate.

**Comment by AnneliesVlaar** [comment]
Hi @ToniSkulj this is on @MarleenGilsing 's wishing list for a long time. Could you take a look if this is doable? If you forsee that it's a big project instead of small let us know! 

Also if you have any question's or suggestions just let us know!

**Comment by ToniSkulj** [comment]
@AnneliesVlaar, sorry for the late response! I will take a look and I will let you know before the end of the weekend.
